it/audit.sh

```diff
@@ -22,7 +22,10 @@ fi
 
 # Check for warnings
 # Ugly fix for SQ 9.8+ that has deprecated login/password authentication but still allows it.
-if grep -iv "propert.* 'sonar.password' .* deprecated" /tmp/scanner.log | grep -q "^WARN: "
+# Also ignore expected warnings regarding k8s.yaml
+if grep -iv "propert.* 'sonar.password' .* deprecated" /tmp/scanner.log \
+ | grep -v "Cannot save .* k8s.yml, ignoring" \
```

Comment:
Implemented this way, I'm not sure this is relevant. Better put the `k8s.yml` file as a unit test to check the `UnsupportedOperationException`.

Reply:
Thanks for the update.

The purpose of unit tests is to make sure things are working as expected. This is valid until you discover something... unexpected, in which case you have to provide a fix and incorporate the unexpected as now something you expect.

Anyway, we may keep `k8s.yml` in `it/src`, there is nothing wrong with that.